test-case/manual/check-audio-privacy-during-D3.md

```diff
@@ -0,0 +1,54 @@
+# Verify audio privacy (hard mute) functionality during D3 state
+Verify audio privacy (hard mute) functionality in Linux in D3 state
+
+## Preconditions
+1. The RVP should have active audio endpoints.
+2. The RVP should have a functional hard mute switch or toggle.
+3. Enable BIOS settings:
+   * Microphone Privacy Mode = "HW managed Microphone Privacy"
+   * Enable corresponding codec options, e.g., SNDW#3 [x] for SoundWire.
+   * Enable DMIC option, e.g., DMIC [x] for DMIC
+4. Make sure UCM is up to date
+   * Copy the ucm and ucm2 trees to the alsa-lib configuration directory (usually located in /usr/share/alsa) including symlinks
+   * Reference: https://github.com/alsa-project/alsa-ucm-conf/
+
+## Test Description
+* Verification of audio privacy (hard mute) functionality during suspension in D3 state in linux system.
+* Should not capture audio samples when the audio privacy switch is turned ON during D3 state.
+* Should capture audio samples when the audio privacy switch is turned OFF during D3 state.
+
+## Steps to Execute
+1. System is booted in the OS
+2. Capture audio via 3.5mm jack ports:
+    ```bash
+    arecord -Dhw:0,0 -c 2 -r 48000 -f S16_LE /tmp/test_sample_mute_during_d3.wav -vvv
```

Comment:
`-Dhw:0,0` may or may not be correct. It would be better to go through `pulse` or whatever audio daemon is running.